src/installer/tests/Directory.Build.targets

```diff
@@ -113,6 +113,28 @@
       <TestsOutputDir Condition="'$(TestsOutputDir)' == ''">$(TestsOutputRootDir)$(TestsOutputName)/</TestsOutputDir>
     </PropertyGroup>
   </Target>
+    <!--
+    We are using to best emulate the live version. This should be removed once we start using the live bits.
+    https://github.com/dotnet/runtime/issues/45972 
+  -->
+  <ItemGroup>
+    <KnownFrameworkReference Include="Microsoft.NETCore.App"
+      DefaultRuntimeFrameworkVersion="$(MicrosoftNETCoreAppVersion)"
+      IsTrimmable="true"
+      LatestRuntimeFrameworkVersion="$(MicrosoftNETCoreAppVersion)"
+      RuntimeFrameworkName="Microsoft.NETCore.App"
+      RuntimePackNamePatterns="Microsoft.NETCore.App.Runtime.**RID**"
+      RuntimePackRuntimeIdentifiers="linux-arm;linux-arm64;linux-musl-arm64;linux-musl-x64;linux-x64;osx-x64;rhel.6-x64;tizen.4.0.0-armel;tizen.5.0.0-armel;win-arm;win-arm64;win-x64;win-x86;ios-arm64;ios-arm;ios-x64;ios-x86;tvos-arm64;tvos-x64;android-arm64;android-arm;android-x64;android-x86;browser-wasm"
+      TargetFramework="$(NetCoreAppCurrent)"
+      TargetingPackName="Microsoft.NETCore.App.Ref"
+      TargetingPackVersion="$(MicrosoftNETCoreAppVersion)" />
+
+    <KnownAppHostPack Include="Microsoft.NETCore.App" 
+      AppHostPackNamePattern="Microsoft.NETCore.App.Host.**RID**"
+      AppHostPackVersion="$(MicrosoftNETCoreAppVersion)"
+      AppHostRuntimeIdentifiers="linux-arm;linux-arm64;linux-musl-arm64;linux-musl-x64;linux-x64;osx-x64;rhel.6-x64;tizen.4.0.0-armel;tizen.5.0.0-armel;win-arm;win-arm64;win-x64;win-x86"
+      TargetFramework="$(NetCoreAppCurrent)" />
+  </ItemGroup>
```

Comment:
We only need this block with the test assets. We don't need it for the actual unit tests.
```suggestion
```